.github/workflows/docker-image.yml

```
@@ -4,296 +4,283 @@ on:
-      contents: read
-      packages: write
-      id-token: write
-    outputs:
-      manifest_digest: ${{ steps.create-manifest.outputs.digest }}
-
+    if: github.event_name != 'pull_request'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)